### ISSUE TITLE
'Flowcharts' example: metaArray exception and plots not positioned correctly

### ISSUE THREAD

**Issue by ghylander** [pyqtgraph/pyqtgraph#2684]
### Short description
In the 'Flowcharts' example there are two bugs. First, the use of `pyqtgraph.metaarray` results in an exception that causes the input signal to not be plotted.

Installing the MetaArray library, and changing lines 16 and 52 to use said library fixes the issue.

Second, the output of the gaussian filter should be plotted in the bottom plot. Instead, it's plotted in the top one.

This seems to be caused by a bug in the code. I'm not sure if this is the desired behaviour, but the following lines (pyqtgraph/examples/Flowcharts.py lines 65-67):

```
pw2Node = fc.createNode('PlotWidget', pos=(150, -150))
pw2Node.setPlot(pw2)
pw2Node.setPlotList(plotList)
```

Result in the 'output' bring plotted on the top plot. Changing the lines to (notice how lines 66 and 67 have been swapped):

```
pw2Node = fc.createNode('PlotWidget', pos=(150, -150))
pw2Node.setPlotList(plotList)
pw2Node.setPlot(pw2)
```
 
Correctly plots the 'output' in the bottom plot.

As stated, I don't know if this behaviour is intentional, or there is a deeper bug in the code that causes this unexpected behaviour.

**Comment by j9ac9k** [comment]
The flowcharts module definitely doesn't get as much love/attention as the rest of the library.  

The metaarray module was removed from pyqtgraph some time ago, ... that's embarrassing that we missed the usage of it in the flowchart module tho!

**Comment by j9ac9k** [comment]
The metaarray issue is no longer relevant as it's been removed from pyqtgraph.  The other issue is still relevant.

**Comment by j9ac9k** [comment]
Fixed wit #3060 being merged.  thanks for the fix @ghylander
